<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>106</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>사진열기</string>
   </property>
  </widget>
  <widget class="QLabel" name="origin_image">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>261</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid yello</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cvt_image">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>261</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid yello</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid yello</string>
   </property>
   <property name="text">
    <string>최소값:</string>
   </property>
  </widget>
  <widget class="QLabel" name="orig_min">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid yello</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid yello</string>
   </property>
   <property name="text">
    <string>최대값:</string>
   </property>
  </widget>
  <widget class="QLabel" name="orig_max">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid yello</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cvt_max">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>330</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid yello</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>330</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid yello</string>
   </property>
   <property name="text">
    <string>최대값:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>290</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid yello</string>
   </property>
   <property name="text">
    <string>최소값:</string>
   </property>
  </widget>
  <widget class="QLabel" name="cvt_min">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid yello</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>image_open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>image_open()</slot>
 </slots>
</ui>
